src/ixdat/constants.py

```diff
@@ -26,15 +26,105 @@
 STANDARD_PRESSURE = 1e5  # Standard pressure of 1 bar in [Pa]
 
 # molecule properties (should probably come from elsewhere).
+
 DYNAMIC_VISCOSITIES = {
-    "O2": 2.07e-05,
-    "N2": 1.79e-05,
-    "Ar": 2.27e-05,
-    "He": 1.99e-05,
-    "CO": 1.78e-05,
-    "CO2": 1.503e-05,
-    "H2": 8.90e-06,
-}  # in [Pa*s]
+    "O2": {
+        200: 14.72e-06,
+        220: 15.98e-06,
+        240: 17.2e-06,
+        260: 18.38e-06,
+        280: 19.53e-06,
+        300: 20.65e-06,
+        320: 21.74e-06,
+        340: 22.8e-06,
+        360: 23.84e-06,
+        400: 25.84e-06,
+        500: 30.49e-06,
+        600: 34.73e-06,
+        700: 38.65e-06,
+        800: 42.33e-06,
+    },
+    "N2": {
+        200: 12.91e-06,
+        220: 13.97e-06,
+        240: 15.00e-06,
+        260: 15.99e-06,
+        280: 16.96e-06,
+        300: 17.89e-06,
+        320: 18.80e-06,
+        340: 19.68e-06,
+        360: 20.55e-06,
+        400: 22.21e-06,
+        500: 26.06e-06,
+        600: 29.58e-06,
+        700: 32.83e-06,
+        800: 35.89e-06,
+    },
+    "Ar": {
+        273.15: 2.1e-05,
+        293.15: 2.23e-05,
+        298.15: 2.27e-05,
+        323.15: 2.42e-05,
+        373.15: 2.73e-05,
+        473.15: 3.28e-05,
+        573.15: 3.77e-05,
+        673.15: 4.22e-05,
+        773.15: 4.64e-05,
+        873.15: 5.04e-05,
+    },
+    "He": {
+        273.15: 1.87e-05,
+        293.15: 1.96e-05,
+        323.15: 2.10e-05,
+        373.15: 2.32e-05,
+        473.15: 2.73e-05,
+        573.15: 3.12e-05,
+        673.15: 3.48e-05,
+        773.15: 3.84e-05,
+        873.15: 4.18e-05,
+    },
+    "CO": {
+        273.15: 1.66e-05,
+        293.15: 1.74e-05,
+        323.15: 1.88e-05,
+        373.15: 2.10e-05,
+        473.15: 2.52e-05,
+        573.15: 2.90e-05,
+        673.15: 3.25e-05,
+        773.15: 3.56e-05,
+        873.15: 3.86e-05,
+    },
+    "CO2": {
+        220: 11.06e-06,
+        240: 12.07e-06,
+        250: 12.57e-06,
+        260: 13.06e-06,
+        280: 14.05e-06,
+        300: 15.02e-06,
+        320: 15.98e-06,
+        340: 16.93e-06,
+        360: 17.87e-06,
+        400: 19.7e-06,
+        450: 21.9e-06,
+        500: 24.02e-06,
+        600: 28e-06,
+        650: 29.87e-06,
+        850: 36.71e-06,
+        1050: 42.69e-06,
+    },
+    "H2": {
+        273.15: 0.84e-05,
+        293.15: 0.88e-05,
+        323.15: 0.94e-05,
+        373.15: 1.04e-05,
+        473.15: 1.21e-05,
+        573.15: 1.37e-05,
+        673.15: 1.53e-05,
+        773.15: 1.69e-05,
+        873.15: 1.84e-05,
+    },
+}  # in [Pa*s] engineeringtoolbox.com
```

Comment:
Great with the sourcing :) Can you maybe move this comment up front and mention the key units

Reply:
I hope I have put the comment the correct place.

Reply:
:D it wasn't that I was commenting on your comment placement, it was just that when the dict was small it didn't really matter where it was, but now that it is so many lines, it makes more sense with it before. In any case, the units in the temperature was the important stuff.